third_party/lua/luadebug.lua

```diff
@@ -97,11 +98,19 @@ local stack_top = 0
 -- Changed using the up/down commands
 local stack_inspect_offset = 0
 
--- Default dbg.read function
-local function dbg_read(prompt)
-    dbg.write(prompt)
-    io.flush()
-    return io.read()
+-- Tarantool console compatible readline support.
+local function dbg_readline(prompt)
+    local console = require 'console.lib'
```

Comment:
From the commit message:

> now they kept visible as 'console_internal'.

Does you mean `console.lib`?

Reply:
I would add that it is changed in c432e9e918f1caa69a3e6050830c9f2734217ba7.

Reply:
Fixed:
>   Implemented readline history and autocomplete by reusing
  readline facilities of Tarantool console. They used to be
  being hidden once Lua 'console' module is loaded. With c432e9e 
  (lua: don't use public module name as internal one), now they 
  are available as 'console.lib'.